scripts/cast-on-tenderly/index.js

```diff
@@ -0,0 +1,87 @@
+import 'dotenv/config';
+import axios from 'axios';
+import { Contract, ethers } from 'ethers';
+
+const NETWORK_ID = 5;
+const CHIEF_ADDRESS = '0x33Ed584fc655b08b2bca45E1C5b5f07c98053bC1';
```

Comment:
Wouldn't it be better to have the `Chainlog` address hard-coded here and fetch `MCD_CHIEF` from it?

Reply:
I would be the same thing in the end (one hardcoded address on top of the file), but with more extra code to review. But we can do it, to make it a little bit less fragile and more robust to updates of the protocol

Reply:
Now only using hardcoding address of the chainlog